de_web_plugin.cpp

```diff
@@ -218,6 +219,7 @@ static const SupportedDevice supportedDevices[] = {
     { VENDOR_SUNRICHER, "ICZB-RM", silabs2MacPrefix }, // iCasa remote
     { VENDOR_SUNRICHER, "ZGRC-KEY", emberMacPrefix }, // Sunricher wireless CCT remote
     { VENDOR_SUNRICHER, "ZG2833K", emberMacPrefix }, // Sunricher remote controller
+    { VENDOR_SUNRICHER, "Motor Controller", sunricherMacPrefix }, // Sunricher window covering controller
```

Comment:
Is this needed?  The controller doesn't seem to have any cluster that's exposed as a `/sensors` resource.

Reply:
You are right, this is not needed for the On/Off cluster. I was not sure about the others, though.